### ISSUE TITLE
NTR: Urinary tract rupture (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8996]
New term request Urinary tract rupture (elthat_100530013913): Bladder  UMLS: 0840564

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000662
- label: Bladder rupture
- parents: [Abnormal bladder morphology (HP:0025487)](https://hpo.jax.org/app/browse/term/HP:0025487)
- pmid: PMID:29262195
- def: A perforation in the wall of the urinary bladder. Bladder rupture, a relatively rare condition, is most commonly due to abdominal or pelvic trauma but may be spontaneous or iatrogenic in association with surgical or endoscopic procedures. In most cases, patients with bladder rupture have gross hematuria. Other symptoms of bladder rupture include pelvic pain, lower abdominal pain, and difficulty voiding.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8996
- synonyms:
